c++/src/capnp/rpc.capnp

```diff
@@ -939,6 +946,11 @@ struct CapDescriptor {
   #
   # Keep in mind that `ExportIds` in a `CapDescriptor` are subject to reference counting.  See the
   # description of `ExportId`.
+  #
+  # Note that it is currently not possible to include a broken capability in the CapDescriptor
+  # table.  Instead, create a new export (`senderHosted`) for each broken capability and then
```

Comment:
Good point. My implementation exported it as hosted because it wasn't a promise, and had an exception to allow far-refs to resolve to exceptions. But far-refs only need to break when the connection is lost, not through Resolve messages. I've fixed my code and updated the PR.